automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -94,37 +95,21 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
       val role = "bigquery.jobUser"
 
       val user: Credentials = UserPool.chooseStudent
-      val userToken: AuthToken = user.makeAuthToken()
-
-      // user is not a project owner
-      val errorMsg1 = "You must be a project owner"
-
-      withBillingProject("auto-goog-role") { projectName =>
-        val addEx = intercept[APIException] {
-          Orchestration.billing.addGoogleRoleToBillingProjectUser(projectName, user.email, role)(userToken)
-        }
-        addEx.getMessage should include(errorMsg1)
-
-        val removeEx = intercept[APIException] {
-          Orchestration.billing.removeGoogleRoleFromBillingProjectUser(projectName, user.email, role)(userToken)
-        }
-        removeEx.getMessage should include(errorMsg1)
-      }(ownerToken)
-
-      // owner is not an owner of this project
-      val errorMsg2 = "The caller does not have permission"
 
-      val otherProject = "broad-dsde-dev"
+      val errorMsg = "You must be a project owner"
+      val unownedProject = "broad-dsde-production"
```

Comment:
If something changed and the test failed, we could actually side-effect broad-dsde-procuction. I think we should use some other project to avoid the chance of doing that.

If there are side-effects due to a test failure, I guess you should probably consider having to undo those... though you might not have permissions to do so. Might be best to create a project in the test case.

The more I suggest that and talk about test runtime (as in this morning's meeting), the more I want to build something special for billing projects so we can reuse them for multiple test cases.

Reply:
Updated to not use `broad-dsde-production` - instead uses `broad-dsde-dev` and two non-owners.